### ISSUE TITLE
CMake issue

### ISSUE THREAD

**Issue by pierrotm777** [raspberrypi/pico-sdk#1432]
Hello,

I try to install the new version 1.5.1 after my old 1.5.0 uninstalled.
I obtain always this error:
`Unable to resolve configuration with compilerPath: "c:/program files/raspberry pi/pico sdk v1.5.0/gcc-arm-none-eabi/bin/arm-none-eabi-g++.exe"`

If i try to compile my project, i have this error:
```
[variant] Loaded new set of variants
[kit] Successfully loaded 1 kits from C:\Users\pierrot\AppData\Local\CMakeTools\cmake-tools-kits.json
[kit] Successfully loaded 1 kits from c:\Users\pierrot\Documents\Pico-v1.5.1\pico-sdk\oXs_on_RP2040-pm\.vscode\cmake-kits.json
[proc] Executing command: "C:\Program Files\Raspberry Pi\Pico SDK v1.5.1\cmake\bin\cmake.EXE" --version
[proc] Executing command: "C:\Program Files\Raspberry Pi\Pico SDK v1.5.1\gcc-arm-none-eabi\bin\arm-none-eabi-gcc.exe" -v
[main] Configuring project: oXs_on_RP2040-pm 
[proc] Executing command: "C:\Program Files\Raspberry Pi\Pico SDK v1.5.1\cmake\bin\cmake.EXE" --no-warn-unused-cli "-DCMAKE_MODULE_PATH:STRING=C:/Program Files/Raspberry Pi/Pico SDK v1.5.1/pico-sdk-tools" -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=FALSE -DCMAKE_BUILD_TYPE:STRING=Debug "-DCMAKE_C_COMPILER:FILEPATH=C:\Program Files\Raspberry Pi\Pico SDK v1.5.1\gcc-arm-none-eabi\bin\arm-none-eabi-gcc.exe" "-DCMAKE_CXX_COMPILER:FILEPATH=C:\Program Files\Raspberry Pi\Pico SDK v1.5.1\gcc-arm-none-eabi\bin\arm-none-eabi-g++.exe" -SC:/Users/pierrot/Documents/Pico-v1.5.1/pico-sdk/oXs_on_RP2040-pm -Bc:/Users/pierrot/Documents/Pico-v1.5.1/pico-sdk/oXs_on_RP2040-pm/build -G Ninja
[cmake] Not searching for unused variables given on the command line.
[cmake] -- Configuring incomplete, errors occurred!
[cmake] Using PICO_SDK_PATH from environment ('..\..\pico-sdk')
[cmake] CMake Error at pico_sdk_import.cmake:63 (message):
[cmake]   Directory 'C:/Users/pierrot/Documents/Pico-v1.5.1/pico-sdk/pico-sdk' not
[cmake]   found
[cmake] Call Stack (most recent call first):
[cmake]   CMakeLists.txt:8 (include)
[cmake] 
[cmake] 
[proc] The command: "C:\Program Files\Raspberry Pi\Pico SDK v1.5.1\cmake\bin\cmake.EXE" --no-warn-unused-cli "-DCMAKE_MODULE_PATH:STRING=C:/Program Files/Raspberry Pi/Pico SDK v1.5.1/pico-sdk-tools" -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=FALSE -DCMAKE_BUILD_TYPE:STRING=Debug "-DCMAKE_C_COMPILER:FILEPATH=C:\Program Files\Raspberry Pi\Pico SDK v1.5.1\gcc-arm-none-eabi\bin\arm-none-eabi-gcc.exe" "-DCMAKE_CXX_COMPILER:FILEPATH=C:\Program Files\Raspberry Pi\Pico SDK v1.5.1\gcc-arm-none-eabi\bin\arm-none-eabi-g++.exe" -SC:/Users/pierrot/Documents/Pico-v1.5.1/pico-sdk/oXs_on_RP2040-pm -Bc:/Users/pierrot/Documents/Pico-v1.5.1/pico-sdk/oXs_on_RP2040-pm/build -G Ninja exited with code: 1
[main] Configuring project: oXs_on_RP2040-pm 
[proc] Executing command: "C:\Program Files\Raspberry Pi\Pico SDK v1.5.1\cmake\bin\cmake.EXE" --no-warn-unused-cli "-DCMAKE_MODULE_PATH:STRING=C:/Program Files/Raspberry Pi/Pico SDK v1.5.1/pico-sdk-tools" -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=FALSE -DCMAKE_BUILD_TYPE:STRING=Debug "-DCMAKE_C_COMPILER:FILEPATH=C:\Program Files\Raspberry Pi\Pico SDK v1.5.1\gcc-arm-none-eabi\bin\arm-none-eabi-gcc.exe" "-DCMAKE_CXX_COMPILER:FILEPATH=C:\Program Files\Raspberry Pi\Pico SDK v1.5.1\gcc-arm-none-eabi\bin\arm-none-eabi-g++.exe" -SC:/Users/pierrot/Documents/Pico-v1.5.1/pico-sdk/oXs_on_RP2040-pm -Bc:/Users/pierrot/Documents/Pico-v1.5.1/pico-sdk/oXs_on_RP2040-pm/build -G Ninja
[cmake] Not searching for unused variables given on the command line.
[cmake] CMake Error at pico_sdk_import.cmake:63 (message):
[cmake]   Directory 'C:/Users/pierrot/Documents/Pico-v1.5.1/pico-sdk/pico-sdk' not
[cmake]   found
[cmake] Call Stack (most recent call first):
[cmake]   CMakeLists.txt:8 (include)
[cmake] 
[cmake] 
[cmake] -- Configuring incomplete, errors occurred!
[proc] The command: "C:\Program Files\Raspberry Pi\Pico SDK v1.5.1\cmake\bin\cmake.EXE" --no-warn-unused-cli "-DCMAKE_MODULE_PATH:STRING=C:/Program Files/Raspberry Pi/Pico SDK v1.5.1/pico-sdk-tools" -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=FALSE -DCMAKE_BUILD_TYPE:STRING=Debug "-DCMAKE_C_COMPILER:FILEPATH=C:\Program Files\Raspberry Pi\Pico SDK v1.5.1\gcc-arm-none-eabi\bin\arm-none-eabi-gcc.exe" "-DCMAKE_CXX_COMPILER:FILEPATH=C:\Program Files\Raspberry Pi\Pico SDK v1.5.1\gcc-arm-none-eabi\bin\arm-none-eabi-g++.exe" -SC:/Users/pierrot/Documents/Pico-v1.5.1/pico-sdk/oXs_on_RP2040-pm -Bc:/Users/pierrot/Documents/Pico-v1.5.1/pico-sdk/oXs_on_RP2040-pm/build -G Ninja exited with code: 1

```

What i can do for the compiler use the new path ?
Thanks for your help,
Pierre

**Comment by kilograham** [comment]
Question best asked on the forums https://forums.raspberrypi.com/viewforum.php?f=143

but these lines

```
[cmake] Using PICO_SDK_PATH from environment ('..\..\pico-sdk')
[cmake] CMake Error at pico_sdk_import.cmake:63 (message):
[cmake]   Directory 'C:/Users/pierrot/Documents/Pico-v1.5.1/pico-sdk/pico-sdk' not
[cmake]   found
```

would definitely seem to suggest your PICO_SDK_PATH Is set incorrectly; better to use an absolute path.

**Comment by pierrotm777** [comment]
Ok, i understand but i don't know witch command i need to do.
My Pico SDK is in this path:
C:\Program Files\Raspberry Pi\Pico SDK v1.5.1
My projects folder is in the same path that the examples files:
C:\Users\pierrot\Documents\Pico-v1.5.1

I have tried to add PICO_SDK_PATH and ..\..\pico-sdk into my PATH system without success.
I have also tried to add this same path into Vscode in the CMake Environnement settings without success.

What good solution the the best ?

Thanks,

Pierre
